sass/components/_main-menu.scss

```diff
@@ -163,3 +123,11 @@
     margin-top: 0;
   }
 }
+
+// remove white borders on field focus
+input:focus,
+select:focus,
+textarea:focus,
+button:focus {
+  outline: none;
```

Comment:
Do you provide any other visual indication of these elements being focused?  
It's encouraged to allow users to easily use the application without mouse and focusable items should always provide visual indication when they are actually focused.